src/Carousel.js

```diff
@@ -160,7 +162,7 @@ class Carousel extends Component {
                         onTouchStart={ this.handleSliderTouchStart }
                         onTouchMove={ this.handleSliderTouchMove }
                         onTouchEnd={ this.handleSliderTouchEnd } >
-                        {React.Children.map(children, (child, i) => (
+                        {nonNullChildren.map((child, i) => (
```

Comment:
```suggestion
                        { nonNullChildren.map((child, i) => (
```

Reply:
I'm surprised this isn't throwing a eslint error. It should. Are there other places like this?

Reply:
I don't see any. Could it be because it's a render interpolation and not a prop? 🤔